src/ts-node.ts

```diff
@@ -97,7 +98,8 @@ function readConfig (options: Options, cwd: string, ts: TSCommon) {
       inlineSources: true,
       declaration: false,
       noEmit: false
-    }
+    },
+    options.compilerOptions
```

Comment:
You can't be allowed to override these options, you should put it between our options and the config options.